Assets/Resources/Shaders/DoubleSided/Double Sided Emissive Diffuse.shader

```diff
@@ -0,0 +1,99 @@
+Shader "Ciconia Studio/Double Sided/Emissive/Diffuse" {
```

Comment:
https://assetstore.unity.com/packages/vfx/shaders/double-sided-shaders-23087

Reply:
We can't use assets from asset store, even if they are free. It's forbidden by asset store license.

It's because we are re-distributing these shaders.